run.py

```diff
@@ -138,6 +148,56 @@ def perform_tests(test_path, test_postfix):
     return ""
 
 
+def perform_nightly_tests(test_path, test_postfix):
```

Comment:
instead of doubling the code you can and flag "nightly", to def perform_test

Reply:
Perform_nightly_tests function removed, add test flags to perform_test